src/terrain/clients/keycloak.clj

```diff
@@ -6,7 +6,7 @@
 (defn- keycloak-url
   "Builds a Keycloak URL with the given path components."
   [& components]
-  (str (apply curl/url (config/keycloak-base-uri) "auth" "realms" (config/keycloak-realm) components)))
+  (str (apply curl/url (config/keycloak-base-uri) "realms" (config/keycloak-realm) components)))
```

Comment:
The `auth` component was already in the configuration values files, so I just removed it here. I thought it was better to try to keep the configuration settings consistent between Terrain and Sonora than to avoid the code change.